kratos/processes/apply_ray_casting_process.h

```diff
@@ -99,16 +102,17 @@ class KRATOS_API(KRATOS_CORE) ApplyRayCastingProcess : public Process
      */
     ApplyRayCastingProcess(
         FindIntersectedGeometricalObjectsProcess& TheFindIntersectedObjectsProcess,
-        const double RelativeTolerance);
+        Parameters ThisParameters = Parameters());
 
-    /**
+	/**
      * @brief Construct a new Apply Ray Casting Process object using an already created search strucutre
      *
      * @param TheFindIntersectedObjectsProcess reference to the already created search structure
      * @param RelativeTolerance user-defined relative tolerance to be multiplied by the domain bounding box size
      * @param pDistanceVariable user-defined variabe to be used to read and store the distance to the skin
      * @param rDistanceDatabase enum value specifying the database from which the distance variable is retrieved (see DistanceDatabase)
      */
+    KRATOS_DEPRECATED_MESSAGE("Deprecated constructor, please use the one with Parameters.")
```

Comment:
OK but this will throw a warning at compile time, which is not useful at all to warn the users during the deprecation period. Hence,  I'd also throw the warning in the constructor body, so the user can see it when running the problem.

Reply:
ok, done